hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/balancer/TestContainerBalancer.java

```diff
@@ -589,6 +593,23 @@ public void balancerShouldFollowExcludeAndIncludeDatanodesConfigurations() {
     }
   }
 
+  @Test
+  public void testContainerBalancerConfiguration() {
+    OzoneConfiguration ozoneConfiguration = new OzoneConfiguration();
+    ozoneConfiguration.set("ozone.scm.container.size", "5GB");
+    ozoneConfiguration.setDouble(
+        "hdds.container.balancer.utilization.threshold", 0.01);
+
+    ContainerBalancerConfiguration cbConf =
+        ozoneConfiguration.getObject(ContainerBalancerConfiguration.class);
+    Assert.assertEquals(cbConf.getThreshold(), 0.01d, DELTA);
+
```

Comment:
can we remove DELTA and use assertTrue(Doublu.compare(a,b) == 0)?

Reply:
I have removed DELTA and use a similar comparison as https://github.com/apache/ozone/blob/8b4d4a9c0421f68d8388f43496bf481eb59d8c62/hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/ozone/container/placement/TestContainerPlacement.java#L76
, please have a check.